lib/routes/routeBackbeat.js

```diff
@@ -508,23 +515,22 @@ function putMetadata(request, response, bucketInfo, objMd, log, callback) {
         if (err) {
             return callback(err);
         }
+
         let omVal;
+
         try {
             omVal = JSON.parse(payload);
         } catch {
-            // FIXME: add error type MalformedJSON
             return callback(errors.MalformedPOSTRequest);
         }
+
         const { headers, bucketName, objectKey } = request;
-        // check if it's metadata only operation
+
         if (headers['x-scal-replication-content'] === 'METADATA') {
             if (!objMd) {
-                // if the target does not exist, return an error to
-                // backbeat, who will have to retry the operation as a
-                // complete replication
                 return callback(errors.ObjNotFound);
             }
-            // use original data locations and encryption info
```

Comment:
these comments do not seem useless at all, they should be kept

Reply:
The one for location and encryption don't add any information ? Reading the code is enough to understand that we reuse the objMd ?

For the other one, the comment don't help to understand the code neither, he just describe what happens later. IMO explaining what happens in other component just add noise. If we do that everywhere, we'll have comment everywhere. A comment should be here to help to understand the code, some weird behaviour or exception ?